components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/constants/AccountConstants.java

```diff
@@ -37,8 +37,8 @@
     public static final String LOGIN_FAIL_TIMEOUT_RATIO_PROPERTY = "account.lock.handler.login.fail.timeout.ratio";
     public static final String NOTIFICATION_INTERNALLY_MANAGE = "account.lock.handler.notification.manageInternally";
 
-    public static final String ACC_LOCKED = "accountlock";
```

Comment:
Do not remove the existing constant. If these are not required, deprecate them and introduce the new constants.

Reply:
This was added by me in a previous pr[1] by removing existing constant. By this PR I have reverted that for backward compatibility

[1].https://github.com/wso2-extensions/identity-event-handler-account-lock/pull/57/files#diff-2266c55ad2d9490ea4faa3e2a83464e7R40